pkg/network/ovn_kubernetes_test.go

```diff
@@ -2118,6 +2118,8 @@ func TestRenderOVNKubernetesEnableIPsec(t *testing.T) {
 	// bootstrap also represents current status
 	// the current cluster is single-stack and has version 1.9.9
 	bootstrapResult := fakeBootstrapResult()
+	bootstrapResult.Infra = bootstrap.InfraStatus{}
+	bootstrapResult.Infra.MachineConfigClusterOperatorReady = true
```

Comment:
Don't you want to test the transition of `MachineConfigClusterOperatorReady` from  `false` to `true` and then back from `true` to `false` as the machine config is applied?

Reply:
sure, added logic to cover mco state transitions.